python/GangaLHCb/__init__.py

```diff
@@ -85,43 +85,21 @@ def _store_root_version():
 
 
 def _store_dirac_environment():
+    from GangaDirac.Lib.Utilities.DiracUtilities import write_env_cache
     diracversion = _guess_version('LHCBDIRAC')
-    import tempfile
-    import subprocess
-    import os
     platform = os.environ['CMTOPT']
     setup_script = 'SetupProject.sh'
```

Comment:
If you mean `lb-run` vs `SetupProject` we should migrate to `lb-run` as this takes care of all situations past and present for us it reverts to `SetupProject` silently and correctly when needed.